js/ajaxTask.js

```diff
@@ -0,0 +1,34 @@
+function send() {
```

Comment:
You may want to add back the js validations so that invalid data is not sent to the server. We skipped them initially in order to focus on the main purpose of the ajax task.